src/main/java/org/apache/ibatis/jdbc/SqlRunner.java

```diff
@@ -250,13 +229,7 @@ private void setParameters(PreparedStatement ps, Object... args) throws SQLExcep
       }
       return list;
     } finally {
-      if (rs != null) {
-        try {
-          rs.close();
-        } catch (Exception e) {
-          // ignore
-        }
-      }
+      // rs should't be close in here,it should be close in caller
```

Comment:
It seems that the `ResultSet` is not closed at all.
Please close the `ResultSet` in the callers or revert this change.

Let me know if I am missing something.

Reply:
I found SqlRunner#insert invoked SqlRunner#getResults and  the ResultSet has been closed in SqlRunner#insert .
SqlRunner#selectAll has the same situation , too. So, in my opinion, ResultSet should be closed in caller.